gnocchi/indexer/alembic/versions/04eba72e4f90_rename_ck_started_before_ended.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 The Gnocchi Developers
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+"""rename ck_started_before_ended
+
+Revision ID: 04eba72e4f90
+Revises: 1e1a63d3d186
+Create Date: 2019-10-01 11:19:38.865522
+
+"""
+
+from alembic import op
+
+from oslo_db import exception
+
+# revision identifiers, used by Alembic.
+revision = '04eba72e4f90'
+down_revision = '1e1a63d3d186'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    bind = op.get_bind()
+
+    if bind and bind.engine.name != 'mysql':
```

Comment:
So there's no need to delete those in MySQL?

Reply:
I decided to change this code if a check constraint with the original name was detected.